libs/core/src/lib/multi-input/multi-input-mobile/multi-input-mobile.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-template let-dialog let-dialogConfig="dialogConfig" #dialogTemplate>
-    <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog">
+    <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog" role="dialog">
```

Comment:
This role comes with mandatory labelling. Please check: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role